src/commands/moderation/clearwarn.ts

```diff
@@ -2,9 +2,9 @@ import { CommandCategories, CommandDefinition, createErrorEmbed } from '../index
 import { createEmbed } from '../../lib/embed';
 import Warn from '../../schemas/warn';
 
-export const clearwarns: CommandDefinition = {
-    names: ['clearwarns'],
-    description: 'Clears the specified number of warns a user has. `Arguments: <id> <count>`',
+export const clearwarn: CommandDefinition = {
+    names: ['clearwarn'],
+    description: 'Clears a warn the user has with the specified index (1-indexed). `Arguments: <id> <index>`',
```

Comment:
Change the command to only require an ID to be provided, meaning only one argument is needed for this command.

For example, `.clearwarn 1341`